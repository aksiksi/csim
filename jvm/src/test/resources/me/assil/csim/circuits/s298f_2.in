10101010101010101
01011110000000111
11111000001111000
11100001110001100
01111011110000000
