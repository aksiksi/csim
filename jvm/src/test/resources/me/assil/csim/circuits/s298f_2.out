00000010101000111000
00000000011000001000
00000000001111010010
00000000100100100101
11111011110000101101
